config.go

```diff
@@ -75,11 +75,15 @@ func newConfig() (Config, []error) {
 	}
 
 	// bools
-	err = input.ValidateWithOptions(os.Getenv("reset_repository"), "yes", "no")
+	resetRepo := os.Getenv("reset_repository")
+	if resetRepo == "Yes" || resetRepo == "No" {
+		log.Warnf("\nInput values 'Yes' and 'No' for reset_repository are DEPRECATED in favor of 'yes' and 'no', these value options will be removed in Git Clone step version 4.1.0\n")
```

Comment:
for reset_repository are -> for reset_repository input are

these value options will be removed in Git Clone step version 4.1.0 -> these value options will be removed in version 4.1.0